### PR TITLE
Gig bulk create

### PR DESCRIPTION
**Pull request by inderps**
POST http://api.cirro.io/v1/bulk/gigs

```
{
  "data": {
    "attributes": {
      "title": "North by Northwest",
      "description": "May the Force be with you.",
      "url": "http://effertz-ledner.com/cleo.heaney",
      "automatic_invites": true,
      "archive_at": "2022-01-31T01:00:00.000Z",
      "total_seats": 5
    },
    "relationships": {
      "worker_invitation_filter": {
        "filter_query": "{ \"size.h\": { \"$lt\": 15 }, \"size.uom\": \"in\", \"status\": \"D\" }"
      },
      "gig_tasks": [
        {
          "title": "Fargo",
          "base_price": 200
        },
        {
          "title": "In the Name of the Father",
          "base_price": 200
        }
      ]
    }
  }
}

```

### CODE REVIEW

**Comment by inderps** [comment]
@fractaledmind can you add pierre to this repo so that he can review too

File: lib/cirro_io/client/response_debugging_middleware.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ module CirroIO
   module Client
     class ResponseDebuggingMiddleware < Faraday::Response::Middleware
       def on_complete(env)
-        # binding.pry
+        binding.pry # rubocop:disable Lint/Debugger
```
Comment (fractaledmind) [inline lib/cirro_io/client/response_debugging_middleware.rb:6]: Is this a leftover artifact?

**Review by fractaledmind — APPROVED** [review]

File: lib/cirro_io/client/response_debugging_middleware.rb (hunk 1 above)
Comment (inderps) [reply lib/cirro_io/client/response_debugging_middleware.rb:6]: Actually i added this middleware for writing tests. https://github.com/test-IO/cirro-ruby-client/blob/master/lib/cirro_io/client/base.rb#L19
So the idea is when I want to write test, i enable this middleware and run the test against actual API. Here in pry i get the response and then put that response in json files below
https://github.com/test-IO/cirro-ruby-client/tree/master/spec/fixtures

Comment (inderps) [reply lib/cirro_io/client/response_debugging_middleware.rb:6]: Maybe its worth to checkup how does vcr gem behaves now. In past we had lot of trouble with maintaining cassettes

**Review by thebluber — APPROVED** [review]

**Pull request merged**